lib/hw_unique_key/hw_unique_key.c

```diff
@@ -82,11 +87,17 @@ int hw_unique_key_derive_key(enum hw_unique_key_slot kmu_slot,
 	uint8_t const *label, size_t label_size,
 	uint8_t *output, uint32_t output_size)
 {
+	int err;
```

Comment:
blank line? I think the compliance script checks this.

Reply:
It doesn't since I passed compliance but I will add it anyway :)